gpt4all-bindings/python/gpt4all/gpt4all.py

```diff
@@ -203,55 +208,78 @@ def generate(
             n_batch: Number of prompt tokens processed in parallel. Larger values decrease latency but increase resource requirements.
             n_predict: Equivalent to max_tokens, exists for backwards compatibility.
             streaming: If True, this method will instead return a generator that yields tokens as the model generates them.
+            callback: A function with arguments token_id:int and response:str, which receives the tokens from the model as they are generated and stops the generation by returning False.
```

Comment:
Correct me if I'm wrong, by the way:  unless I've missed something in the new `LLModel` logic, response should be bytes.